tests/scripts/test-ref-configs.pl

```diff
@@ -57,11 +57,19 @@
 my $config_h = 'include/mbedtls/mbedtls_config.h';
 
 system( "cp $config_h $config_h.bak" ) and die;
-sub abort {
-    system( "mv $config_h.bak $config_h" ) and warn "$config_h not restored\n";
-    # use an exit code between 1 and 124 for git bisect (die returns 255)
+
```

Comment:
Note: this is going to conflict with the PR that removes this script. I'd suggest removing this commit to avoid the conflict.

Reply:
I wanted to get detailed CI feedback here for as long as `test-ref-configs` failed. Changing the script is out of scope here and I'll remove it in the next rebase (there'll be a next rebase once https://github.com/Mbed-TLS/mbedtls/pull/9563 is merged).